celltype_annotation/scripts/render-singleR-report.R

```diff
@@ -39,7 +39,7 @@ option_list <- list(
   make_option(
     opt_str = c("--s3_prefix"),
     type = "character",
-    default = "s3://nextflow-ccdl-results/scpca/processed/results/",
+    default = "s3://nextflow-ccdl-results/scpca/processed/results",
```

Comment:
🎢

Reply:
Just to continue the rollercoaster, I would leave this change, but you should probably not use `file.path()` at https://github.com/AlexsLemonade/sc-data-integration/blob/9e3f10fc94bf6c6da8144622367fef423c5a4b6b/celltype_annotation/scripts/render-singleR-report.R#L67 unless you add `fsep = "/"` since you that path is not a local file path. You don't want to accidentally add a backslash if running on Windows!